06-building-a-program-start-to-finish/writing-tests/output/invertedIndexTests.js

```diff
@@ -0,0 +1,308 @@
+const invertedIndex = new InvertedIndex();
```

Comment:
This file can be named better. There's a way to name test files.

Also, please check the naming of your test suites and specs. They should follow BDD guidelines.